packages/tests/src.ts/test-contract.spec.ts

```diff
@@ -306,8 +307,28 @@ describe("Test Contract Transaction Population", function() {
         assert.strictEqual(Array.isArray(transactions), true);
         assert.strictEqual(transactions.length, 2);
     });
+
+    // TODO: skipped as we should not spam testnet with random contracts
+    // Previewnet is not really a choice as it will (soon or later) result in INVALID_SIGNATURE pre-checks as of cleanup
+    xit("should be able to deploy a contract", async function() {
+        const hederaEoa = {
+            account: "0.0.29559509",
+            privateKey: "0xbb621d187c22459ab6ed6768bd516bd630a087df4d5a4fbe95d77e87af10c6e1"
+        };
+        const provider = ethers.providers.getDefaultProvider('testnet');
+        // @ts-ignore
+        const wallet = new ethers.Wallet(hederaEoa, provider);
+        const abi = JSON.parse(fs.readFileSync('examples/assets/abi/GLDToken_abi.json').toString());
```

Comment:
nit: Instead of using `fs` to load files, we can simply do `import`.
Example:
https://stackoverflow.com/questions/34944099/how-to-import-a-json-file-in-ecmascript-6